src/workloads/issues/MongosLatency.yml

```diff
@@ -0,0 +1,233 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/product-perf"
+Description: |
+  This workload was created to reproduce the issue described in HELP-25757 / SERVER-58997. The goal
+  is to induce periods of latency in the MongoS processes that are not caused by the MongoD processes.
+
+  The repo comprises of:
+    * A sharded collection of 6000 documents with 4 indexes.
+    * The collection is sharded on the _id field.
+    * A workphase of 270 threads that runs for 60 minutes. Each CrudActor thread
+      executes a mixed workload of 40 interleaved operations (20 Finds and 20 Updates).
+
+  The main metrics to monitor are:
+    * The Operation Throughput
+    * The Operation latency
+    * "ss connections active"
+
+Clients:
+  Default:
+    QueryOptions:
+      minPoolSize: 1
+      maxPoolSize: 10000
+
+GlobalDefaults:
+  DbName: &db test
+  Collection: &col Collection0
+  FullName: &fullname test.Collection0
+
+  ClassId: &classId {^ObjectId: {^FormatString: {"format": "c%04x%s", "withArgs":[{^RandomInt: {min: 0, max: 200}}, {^RandomString: {length: 19, alphabet: b}}]}}}
```

Comment:
There's go to be a better way to do this. The alphabet has only one character in it, so this is making the string "bbbbbbbbbbbbbbbbbbb" but generating a fair number of random values along the way. Did you mean for this to be a string of b's?

Reply:
Yeah, I think I was just slavishly copying the source repo (although it appends a). I'll put in the explicit strings for now.